CMakeLists.txt

```diff
@@ -120,8 +127,11 @@ set_target_properties(${APP_NAME} PROPERTIES
                       XCODE_ATTRIBUTE_TARGETED_DEVICE_FAMILY ${DEVICE_FAMILY}
                       MACOSX_BUNDLE_INFO_PLIST ${CMAKE_CURRENT_SOURCE_DIR}/plist.in
                       XCODE_ATTRIBUTE_CLANG_ENABLE_OBJC_ARC YES
-                      XCODE_ATTRIBUTE_COMBINE_HIDPI_IMAGES "NO"
-                      XCODE_ATTRIBUTE_INSTALL_PATH "$(LOCAL_APPS_DIR)")
+                      XCODE_ATTRIBUTE_COMBINE_HIDPI_IMAGES NO
+                      XCODE_ATTRIBUTE_INSTALL_PATH "$(LOCAL_APPS_DIR)"
+                      XCODE_ATTRIBUTE_ENABLE_TESTABILITY YES
+                      XCODE_ATTRIBUTE_GCC_SYMBOLS_PRIVATE_EXTERN YES
```

Comment:
You don't see the same error as me? 

(I attached a screenshot of the error in Xcode back on July 6.

https://user-images.githubusercontent.com/733784/27941801-7fc0d6fc-62a0-11e7-9ed8-ab3322b1266a.png